INSTALL.rst

```diff
@@ -21,13 +21,20 @@ Then, the rest of the packages can be installed via:
 
 ::
 
-    yum install libvirt bridge-utils ipmitool telnet sqlite httpd mod_wsgi python-pip qemu-kvm python-virtinst
+    yum install libvirt bridge-utils ipmitool telnet httpd mod_wsgi python-pip qemu-kvm python-virtinst
```

Comment:
We're pulling in `python-virtinst` for the `virt-clone` in CentOS 7 this has been renamed `virt-install`. I just ran into that on the test environment you set up.